\noindent\begin{minipage}{\textwidth} % to avoid pagebreaks despite longtable environment%\begin{landscape}\begin{singlespacing}\begin{small}\vspace*{-2em}%\setlength{\abovecaptionskip}{-10pt}\begin{longtable}{>{\textsf\bgroup\raggedleft\arraybackslash}p{3cm}<{\egroup} >{\textsf\bgroup}p{5cm}<{\egroup} >{\textsf\bgroup}p{6cm}<{\egroup}} % defining the columns - these must match the widths defined for the mini pages down below!\caption[Biases and artifacts of ChIP-seq data.]{\textsf{Biases and artifacts of ChIP-seq data. Given a rigorously tested antibody, ChIP-seq still suffers from additional technical problems that are due to the sequencing process as well as bioinformatic hurdles.}} \\ % the \\ is important! see http://tex.stackexchange.com/questions/103698/extra-alignment-tab-with-longtable%%%%%%%%%%%%%%%% table title\textbf{Problem} & \textbf{Reasons} & \textbf{Solutions}\tabularnewline%-----------------------\toprule\begin{minipage}{3cm}				%\vskip 6pt					\textbf{Chromatin context} and \textbf{transcription}				%\vskip 4pt			\end{minipage}			&	\begin{minipage}{5cm}				%\vskip 6pt				\begin{itemize}[noitemsep,leftmargin=*]					\item euchromatic chromatin is more easily fragmented					\item formaldehyde fixation does not capture short-lived protein-DNA interactions \citep{Gavrilov2014} and pref\-er\-en\-tially cross\-links proteins with each other \citep{Neill2003}					\item chromatin extraction may unspecifically enrich for highly active genes \citep{Teytelman2013}								\end{itemize}					%\vskip 4pt			\end{minipage}			& \begin{minipage}{6cm}				\vskip 6pt					\begin{itemize}[noitemsep,leftmargin=*]						\item cell-type- and condition-specific input controls \citep{Vega2009, Landt2012}						\item if possible, avoid crosslinking \citep{Neill2003, Kasinathan2014}						\item optimized chromatin extraction including extensive de-crosslinking, RNase and Proteinase treatments \cite{Waldminghaus2010}						\item to identify hyper-ChIPable regions, ChIP against a non-endogenous protein was suggested \citep{Teytelman2013}						\item immunoprecipitation with immunoglobulin G (IgG, mock IP) \citep{Landt2012,Marinov2014}											\end{itemize}					\vskip 4pt			\end{minipage}\tabularnewline  \hline %---------------------------\begin{minipage}{3cm}				%\vskip 6pt					\textbf{Sequencing errors\\and errors in base calling}				%\vskip 4pt			\end{minipage}			& \begin{minipage}{5cm}				\vskip 6pt				\begin{itemize}[noitemsep,leftmargin=*]					\item imperfect sequencing chemistry and signal detection					\item loss of synchronized base-incorporation into the single molecules within one cluster of clonally amplified DNA frag\-ments (phasing and pre-phasing) (see \tref{tab:sequencing})					\item signal intensity decay				\end{itemize}					\vskip 4pt			\end{minipage}				& \begin{minipage}{6cm}				%\vskip 6pt					\begin{itemize}[noitemsep,leftmargin=*]					\item improvement of the sequencing chemistry and detection					\item optimized software for base calling \citep{Ledergerber2011}					\item computational removal of bases with low base calling scores \citep{Minoche2011}					\end{itemize}				%\vskip 4pt				\end{minipage}\tabularnewline  \hline %-----------------------\begin{minipage}{3cm}				%\vskip 6pt					\textbf{GC bias} and\\					\textbf{duplicate reads}				%\vskip 4pt			\end{minipage}			&	\begin{minipage}{5cm}				%\vskip 6pt					\begin{itemize}[noitemsep,leftmargin=*]						\item GC-rich regions are preferably amplified by PCR						\item small fragments are preferably hybridized to the flow cell						\item low number of founder DNA frag\-ments					\end{itemize}				%\vskip 4pt			\end{minipage}			& \begin{minipage}{6cm}				\vskip 6pt					\begin{itemize}[noitemsep,leftmargin=*]						\item optimizing cross-linking, sonication, and the ChIP protocol to ensure that the majority of the genome is present in the sample						\item limiting PCR cycles during library preparation to a minimum						\item computational correction for GC content \citep{Cheung2011, Benjamini2012} and elimination of reads from identical DNA fragments					\end{itemize}					\vskip 4pt			\end{minipage}\tabularnewline  \hline %-----------------------------\begin{minipage}{3cm}				%%\vskip 6pt					\textbf{Copy number\\variations and\\mappability}				%\vskip 4pt			\end{minipage}			&			\begin{minipage}{5cm}				%\vskip 6pt				\begin{itemize}[noitemsep,leftmargin=*]					\item incomplete genome assemblies					\item strain-specific differences to the reference assembly may lead to misrepresentation of individual loci					\item repetitiveness of genomes and shortness of sequencing reads hinder unique read alignment				\end{itemize}					%\vskip 4pt			\end{minipage}			& \begin{minipage}{6cm}				\vskip 6pt					\begin{itemize}[noitemsep,leftmargin=*]										\item increased sequencing depth and control (non-ChIP) sample aid the computational identification of problematic loci \citep{Bailey2013, Chen2012, Jung2014, Landt2012, Kidder2011}										\item longer sequencing reads										\item paired-end sequencing \citep{Chen2012, Bailey2013}										\item exclusion of blacklisted regions that are known to attract artificially high read numbers \citep{Carroll2014, Blacklists}										\item computational correction for mappability \citep{Cheung2011}										\item considering the \textit{effective} genome size \citep{Zhang2008}					\end{itemize}					\vskip 4pt			\end{minipage}\tabularnewline \bottomrule%-----------------------------\label{tab:biases}\end{longtable}\end{small}\end{singlespacing}%\end{landscape}\end{minipage}