% Appendix A

\chapter{Supplemental Information} % Main appendix title
\label{SuppInfo} % For referencing this appendix elsewhere, use \ref{AppendixA}
%\lhead{Supplemental Information} % This is for the header on each page - perhaps a shortened title
\fancyhead[RO,LE]{Appendix: Supplemental Information}
\fancyfoot{}
\fancyfoot[C]{\thepage}

%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Supplemental material related to the MSL and NSL complexes}
%
\begin{figure}[h!]
\includegraphics[width=1\textwidth]{Figures/histoneMarkDistribution.png}
\begin{footnotesize}
\setlength{\abovecaptionskip}{0ex}
\caption[Distribution of histone marks along an active gene.]{\textsf{Schematic patterns of selected histone marks along an active gene. Histone modifications associated with active transcription are shown in green (methylation) and orange (acetylation), repressive histone marks are shown in blue. Active promoters are strongly labelled with H3K9ac and H3K4me3, surrounded by H3K4me2/1. Inactive genes are typically covered with H3K9me2/3, H4K20me3 and H3K27me3 that also peaks around the silenced promoters. The image is based on \citep{Barth2010}. Note that the signal of H4K16ac for dosage compensated genes in \textit{Drosophila} inreases towards the 3'-end.}}
\label{fig:histoneDistribution}
\end{footnotesize}
\end{figure}
%\subsection{Protein names}
\input{Tables/Table_ProteinNames}
%
%\subsection{Cancer-related observations}
\input{Tables/Table_Cancer}
%
%\subsection{Enzymes within the complexes}
\input{Tables/Table_Enzymes}
%
%---------------------------------
%%% Figure NSL targets
\begin{figure}[tb]
	\begin{minipage}[c]{0.50\textwidth}
		\includegraphics[width=\textwidth]{Figures/NSLtargetgenes_scissored.pdf}
	\end{minipage} \hfill
	%%%%%%%%%%
	\begin{minipage}[c]{0.4\textwidth}
		\begin{footnotesize}
			\caption[ChIP-seq signals of MSL complex members for NSL targets and NSL-non-bound genes in \textit{Drosophila}.]{\textsf{ChIP-seq signals of MSL complex members for NSL targets and NSL-non-bound genes in \textit{Drosophila}. All images were generated with \texttt{computeMatrix} and \texttt{heatmapper} from the deepTools suite \citep{Ramirez2014} using the scale-regions mode to scale gene bodies to 2~kb. See \tref{tab:ChIPseqSamplesDmel} and \ref{tab:Becker} for details about the ChIP-seq data sets.
\textbf{A)} There is no difference in the ChIP-seq signals of MOF, H4K16ac and MSL1 from female larva for autosomal (chromosome 2L, chr2L) and X-linked (chrX) genes, but there are almost exclusively strong enrichments for NSL targets.
\textbf{B)} In males, the signals of the MSL complex are visibly stronger for X-linked NSL targets, but only MSL3 and H4K16ac show significant enrichments for non-NSL-bound genes.}}
			\label{fig:NSLtargetgenes}
		\end{footnotesize}
	\end{minipage}
\end{figure}

\clearpage

%---------------------------------

%\input{Tables/Table_Functions01}
%\input{Tables/Table_Functions02}
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Supplemental bioinformatics-related tables}
%
%\subsection{Bioinformatic tools and software}
\input{Tables/Table_Tools}
% Quality Metrics of ChIP-seq experiments
\input{Tables/Table_QScores}
%\subsection{Peak calling}
\input{Tables/Table_PeakCallingStrategies}
%
\subsection{Datasets}
\label{Datasets}
All analyses were enriched by the integration of previously published, publicly available high-throughput sequencing data and annotation.\\
%
\input{Tables/Table_Databases}
%
All samples generated by members of the Akhtar lab were uploaded to public repositories. Accession numbers starting with G indicate the GEO database, accession numbers starting with E indicate ArrayExpress (see \tref{tab:DB}). The DNA read numbers are based on filtered reads. Read and accession numbers refer to all available replicates of each experiment.
%
\subsubsection{\textit{Drosophila} datasets}
%
The subsequently listed data was used for Lam, Mühlpfordt, Vaquerizas et al. \citep{Lam2012} (\aref{SuppPub_NSL}) and Chlamydas et al.\\
\input{Tables/Table_DmelSamplesInHouse}
\clearpage
\input{Tables/Table_DmelSamplesPublic}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsubsection{Mouse datasets}
The following data was used for Chelmicki, Dündar et al.\citep{Chelmicki2014} (\aref{SuppPub_MSL}). MSL1 and input samples were also used for Chlamydas et al. (\aref{SuppPub_MSL1}). \\
\input{Tables/Table_MmSamplesInHouse}
\input{Tables/Table_RNASeqSamples}
\input{Tables/Table_MmSamplesPublic}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
